packages/lite-insight/src/insight-node/api.ts

```diff
@@ -16,6 +16,9 @@ export const getDataInsights = (sourceData: Datum[], options?: InsightOptions):
   return extractResult;
 };
 
+/**
+ * @deprecated since 3.0.0, web workers inside a package has a low compatibility with bundlers.
```

Comment:
```suggestion
 * @deprecated This function will be removed in 3.0.0 because web workers inside a package has a low compatibility with bundlers. The alternatives after 3.0 are xxxxx
```